e2e/test/scenarios/filters-reproductions/filters-reproductions.cy.spec.js

```diff
@@ -1482,3 +1482,42 @@ describe("issue 47887", () => {
     });
   });
 });
+
+describe("Issue 48851", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsNormalUser();
+    cy.viewport(1050, 300);
+  });
+
+  const manyValues = Array(12)
+    .fill(0)
+    .map(() => Math.round(Math.random() * 1000_000_000_000).toString(36))
+    .join(", ");
+
+  function openTableHeaderMenu(name) {
+    cy.findAllByTestId("header-cell")
+      .contains(name)
+      .trigger("mousedown")
+      .trigger("mouseup");
```

Comment:
Why not `click()`? I see this code in other tests:

```suggestion
    cy.findAllByTestId("header-cell").contains(name).click();
```